docs/developer/local-deployement.md

```diff
@@ -0,0 +1,111 @@
+# Local Deployment of Kepler with Kepler-Operator on Kubernetes/OpenShift
```

Comment:
Please note that #189 is about local `development` and not how to deploy kepler locally. 
It is to help new comer start with setting up  a development env so that their changes can be tested easily locally.